nemo_skills/prompt/utils.py

```
@@ -302,6 +302,29 @@ def fill(
+        Returns:
+            int | None: Token count, or None if no tokenizer is set.
+        """
+        if self.tokenizer is None:
+            return None
+
+        if isinstance(messages, str):
+            return len(self.tokenizer.encode(messages, add_special_tokens=False))
+        elif isinstance(messages, list):
+            try:
+                return len(self.tokenizer.apply_chat_template(messages, tokenize=True))
+            except Exception as e:
```

Ruff (0.13.1)

Do not catch blind exception: `Exception`

(BLE001)